src/test/suite/fixtures/recorded/actions/copyVest.yml

```diff
@@ -30,4 +30,5 @@ finalState:
     - contentRange:
         start: {line: 1, character: 6}
         end: {line: 1, character: 11}
+      hasExplicitRange: false
```

Comment:
note that "copy" uses `CommandAction` and doesn't change the document, so it gets rich targets now.  This one loses its explicit range because it is a decorated mark, and they don't have explicit range.  So now you can say "copy air", then "take inside that"